frontend/components/buttons/Button/_styles.scss

```diff
@@ -297,11 +300,6 @@ $base-class: "button";
 
   &--disabled {
     @include disabled;
-
-    &:hover,
-    &:focus {
-      cursor: default;
```

Comment:
Why were you able to drop this?

Reply:
I added it into the `@include disabled`: https://github.com/fleetdm/fleet/pull/28346/files#r2056676244